vision/faceDetection.js

```diff
@@ -64,12 +67,18 @@ function highlightFaces (inputFile, faces, outputFile, Canvas, callback) {
     context.strokeStyle = 'rgba(0,255,0,0.8)';
     context.lineWidth = '5';
 
-    faces.forEach(function (face) {
+    faces.forEach((face) => {
```

Comment:
Don't use an arrow function here if you're not going to convert the whole file to arrow functions.